### DISCUSSION TITLE
Could you export FieldSelectOption?

### DISCUSSION THREAD

**Discussion by dan-leech** [vueform/builder#50] [Ideas]
Could you export from the builder library `FieldSelectOption` and `FieldSelectComponent` which are used in conditions fields selector slots?
I want to use same elements selector in my custom fields.
<img width="364" alt="image" src="https://github.com/vueform/builder/assets/6331634/4b76b12e-a217-47be-aade-b293753e120a">

**Comment by adamberecz** [comment]
It's currently not structured in a way that could be easily exported. However, the [`elementSelector`](https://github.com/vueform/builder/tree/main/plugins/elementSelector) plugin does something very similar, you might be able to use it to recreate it.

Hint: [this](https://github.com/vueform/builder/blob/main/plugins/elementSelector/TextElement.vue#L35) is what actually renders the select box, so [this](https://github.com/vueform/builder/blob/main/plugins/elementSelector/index.mjs#L122) is what you pretty much need from the plugin.
